### PR TITLE
Victoria's Blog Rough Draft

### PR DESCRIPTION
**Pull request by caovicto**

### CODE REVIEW

**Comment by djrain** [comment]
Cool blog post, Victoria! Here's my feedback:

**Intro**:
 
- The eevee gif and example are awesome.
- The introductory sentence is a bit hard to follow.
- It would be nice to have a bit more context for those who aren't super familiar. You could briefly explain or show what a genome is, and when/why are we doing these operations on them? And what is a page fault? 


**Body Sections**:

- Diagrams are great.
- Well written, good spacing and sections. Perhaps a bit more specific explanations would make things even more clear.
- In SegmentNode section, you could format what it contains into a bulleted list. typo - "vairables"
- typos in Faster Access with Indexing Table section - "nunber", "amortixed"


**Results**:

- Obviously, graphs are missing :P


**Take-Aways**:

- I'm not totally sure what your conclusions were. You could sum up the results, and suggest in what situations the new or naive methods would be best.
- typo "implemtations"



Overall, great use of images. A bit more explanation would make it easier to understand your ideas and results. Consider defining terms that might not be known.

**Comment by chavez-ju** [comment]
Very nice Rough Draft! Here's my feedback:

- Possibly include some information on what you would do differently while working on your project that would possibly yield even better results (e.g. choosing different data structures that could yield better performance). This could be added within your conclusion, or in its own header near the conclusion
- Some of the .png files don't seem to be displaying properly, within your SegmentNode header for example (that could just be on my end)
- Possibly add in some figures of your linked list proposed method describing how it works (again, this could possibly be there already, but some of the added files aren't appearing correctly on my end)

Overall, really great job on describing the problem at hand and how your project is here to solve it. Great job on not only specifying the details of your project, but also giving a brief overview of the problem at hand. This really helps alleviate the possibility of confusing the reader (really like the Pokemon pics too haha)
